docs/src/main/docs/guides/91_mp-tutorial.adoc

```diff
@@ -0,0 +1,1267 @@
+    </dependencies>
+
+    <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId> <2>
+                    <artifactId>maven-jar-plugin</artifactId>
+                    <version>2.5</version>
+                    <configuration>
+                        <archive>
+                            <manifest>
+                                <addClasspath>true</addClasspath>
+                                <classpathPrefix>${libs.classpath.prefix}</classpathPrefix>
+                                <mainClass>${mainClass}</mainClass>
+                                <useUniqueVersions>false</useUniqueVersions>
+                            </manifest>
+                        </archive>
+                    </configuration>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId> <3>
+                <executions>
+                    <execution>
+                        <id>copy-dependencies</id>
+                        <phase>prepare-package</phase>
+                        <goals>
+                            <goal>copy-dependencies</goal>
+                        </goals>
+                        <configuration>
+                            <outputDirectory>${copied.libs.dir}</outputDirectory>
+                            <overWriteReleases>false</overWriteReleases>
+                            <overWriteSnapshots>false</overWriteSnapshots>
+                            <overWriteIfNewer>true</overWriteIfNewer>
+                            <overWriteIfNewer>true</overWriteIfNewer>
+                            <includeScope>runtime</includeScope>
+                            <excludeScope>test</excludeScope>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+
+</project>
+---- 
+
+<1> New entries are created in the `properties` to set the name of the `libs`
+ directory where the dependencies will be placed.   
+<2> In the `build` section,
+ a `plugin` entry is added to configure the `maven-jar-plugin` to set the
+ main class in the manifest, and to add the `libs` directory to the
+ manifest classpath.  
+<3> Additionally, the `maven-dependency-plugin` is
+ configured to copy the dependencies into the `libs` directory.
+
+With these updates in place, build the application with Maven:
+
+[source,bash]
+.Build the Application
+----
+mvn package
+----
+
+This will build the application jar and save all runtime 
+dependencies in the `target/libs` directory. This means you can easily start the
+ application by running the application jar file:
+
+[source,bash]
+.Run the application
+----
+java -jar target/helidon-mp-tutorial-1.0-SNAPSHOT.jar
```

Comment:
I _think_ you have the build final name set up to be just `helidon-mp-tutorial.jar`, yes?

Reply:
i did not have that, but i should :(